LibTCPServer.emx

```diff
@@ -366,13 +366,13 @@ srv->start();
 	}&#xD;
 	Object::Ptr object = result.extract&lt;Object::Ptr>();&#xD;
 	Var var_cmd = object->get(&quot;command&quot;);&#xD;
-	string cmd = var_cmd.convert&lt;string>();&#xD;
+	std::string cmd = var_cmd.convert&lt;std::string>();&#xD;
 	if (config.logToStdOut)&#xD;
-		cout &lt;&lt; &quot;JSON command received: &quot; &lt;&lt; cmd &lt;&lt; endl &lt;&lt; flush;&#xD;
```

Comment:
Why did you remove flush. It also comes from std. please, keep it.

Reply:
The following link indicated that endl internally having new line followed by flush. I think adding one more flush is not required. Let me know you view on this.
https://en.cppreference.com/w/cpp/io/manip/endl